### PR TITLE
Avoid traceback when transforming links on content type with no primary field

### PR DESCRIPTION
**Pull request by laulaz**
This refs #51

Other possible implementation : 
```
        field_id = self.request.get('field_id')
        if field_id is None:
            field_id = IPrimaryFieldInfo(self.context).fieldname
```

I have no preference ...

### CODE REVIEW

**Comment by laulaz** [comment]
@jenkins-plone-org please run jobs

File: plone/app/versioningbehavior/browser.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def __call__(self):
         if not version_id:
             raise ValueError(u'Missing parameter on the request: version_id')
 
-        field_id = self.request.get('field_id', IPrimaryFieldInfo(self.context).fieldname)
+        field_id = self.request.get('field_id') or IPrimaryFieldInfo(self.context).fieldname
```
Comment (jensens) [inline plone/app/versioningbehavior/browser.py:107]: Sorry, I do not get how this solves it. What do we get with ``IPrimaryFieldInfo(self.context)``? ``None`` or a ``LookupError``? 

In code I would express it like so:

```suggestion
        field_id = self.request.get('field_id')
        if not field_id:
            primary_field_info = IPrimaryFieldInfo(self.context, None)
            if primary_field_info:
                field_id = primary_field_info.fieldname
            else:
               # What now? 
               raise ValueError("Houston, we have no field_id")
```

Comment (laulaz) [reply plone/app/versioningbehavior/browser.py:107]: > Sorry, I do not get how this solves it. What do we get with `IPrimaryFieldInfo(self.context)`? `None` or a `LookupError`?

We get `TypeError` (Could not adapt) (#51)

I didn't want to change / break existing behaviors, that's why my change is very minimal ...

Comment (jensens) [reply plone/app/versioningbehavior/browser.py:107]: phew. ok, so thats the idea... so, lets go with it for now.

**Pull request merged**